config/locales/doc_auth/zh.yml

```diff
@@ -125,6 +125,7 @@ zh:
       verify_at_post_office: 去邮局验证身份
       verify_identity: 验证你的身份
       verify_online: 在网上验证你的身份
+      verify_online_selfie: 使用手机在网上验证你的身份
```

Comment:
there's a period at the end of this sentence in the translations doc

Reply:
I don't see one in the [translation document](https://docs.google.com/document/d/1vhGmTiF312PjOPWYvJH9oV0X95r7OIgGnQKLDCYGeVI/edit). This is the conditional H2. Also, all other languages for verify_online_selfie do not end in a period. Please reach out again if you disagree after reading this in case I have things mixed up. 

![Screenshot 2024-05-08 at 10 27 34 AM](https://github.com/18F/identity-idp/assets/125507397/3490518c-7e15-4231-9fef-5598b3cd4208)

<img width="544" alt="Screenshot 2024-05-08 at 10 28 43 AM" src="https://github.com/18F/identity-idp/assets/125507397/e70cfb62-e917-42db-855a-3340290a2985">

Reply:
ok sorry i commented on the wrong line